### ISSUE TITLE
Snapshots created after 2021.3.0 are displayed as 8 digit hex number ignoring the text you enter

### ISSUE THREAD

**Issue by KennethLavrsen** [home-assistant/frontend#8533]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I go to Supervisor Snapshots and create a full snapshot of my system. I enter a short simple title text in plain English
The snapshot is created and then displayed as an 8 digit hex number.
I have tried 5 times in two different browsers.
Existing old snapshot are displayed with their correct titles.

### Describe the behavior you expected

I expect to see the title I typed

### Steps to reproduce the issue

I described it above

### What version of Home Assistant Core has the issue?

core-2021.3.0

### What was the last working version of Home Assistant Core?

core-2021.2.3

### In which browser are you experiencing the issue with?

Google Chrome 88.0.4324.182 and up to date IOS Safary

### Which operating system are you using to run this browser?

Ubuntu Linux as well as latest IOS

Ludeus suggested it is a front end bug.

**Comment by ludeeus** [comment]
How are you accessing the website? local IP? http? https? with proxy?

**Comment by KennethLavrsen** [comment]
Update

I compared the content of a correctly create snap and the new that is displayed wrong.

They also download wrong. They are not given any filename at all. They are called .tar which you can edit.

The content of the JSON inside starts with 

```
 "slug": "fd17dc39",
  "name": "",
  "date": "2021-03-03T22:26:27.501858+00:00",
  "type": "full",
  "homeassistant": {
    "boot": true,
    "port": 8123,
    "wait_boot": 600,
    "audio_input": "None",
    "watchdog": true,
    "audio_output": "None",
    "ssl": false,
```

So clearly it is at creation the name you enter is ignored as the JSON shows "name": ""
An old snap shows the correct filename

**Comment by KennethLavrsen** [comment]
Answer to question. I access directly with local IP and no SSL.

**Comment by ludeeus** [comment]
yeah, I only tested with partial, was fixed 30min ago for full in #8535
